pydoctor/test/test_astbuilder.py

```diff
@@ -844,6 +844,26 @@ def __init__(self):
     assert m.docstring == """module-level"""
     assert to_html(get_parsed_type(m)) == '<code>bytes</code>'
 
+@typecomment
+@systemcls_param
+def test_type_comment(systemcls, capsys):
+    mod = fromText('''
+    d = {} # type: Dict[str, int]
+    i = [] # type: ignore[misc]
```

Comment:
Here we are testing that type inference is done when the type comment is an ignore? If that's correct, perhaps mention that in the test docstring?

Reply:
Actually it's only testing that `type: ignore` doesn't lead to an invalid type being reported. I was worried that `ignore` would be reported as the type, but Python 3.8's `ast` module will store `type: ignore` information in a separate per-module list. However, since the interface of `ast` has received incompatible changes before (like how line numbers are handled), I thought it was a good precaution to add a check that verifies that behavior for the future.

I'll add a comment explaining that.